x/tss/keeper/msg_server.go

```diff
@@ -106,6 +106,13 @@ func (s msgServer) HeartBeat(c context.Context, req *types.HeartBeatRequest) (*t
 		return nil, fmt.Errorf("sender [%s] is not a validator", req.Sender)
 	}
 
+	for _, k := range req.KeyIDs {
+		_, ok := s.GetKey(ctx, k)
+		if !ok {
+			return nil, fmt.Errorf("operator '%s' sent heartbeat for unknown key ID '%s'", valAddr.String(), k)
```

Comment:
```suggestion
			return nil, fmt.Errorf("operator %s sent heartbeat for unknown key ID %s", valAddr.String(), k)
```
Just for consistency and easy copy/paste :p